composer/loggers/mlflow_logger.py

```diff
@@ -54,6 +55,7 @@ class MLFlowLogger(LoggerDestination):
             synchronously to the MLflow backend. If ``False``, Mlflow will log asynchronously. (default: ``False``)
         log_system_metrics (bool, optional): Whether to log system metrics. If ``True``, Mlflow will
             log system metrics (CPU/GPU/memory/network usage) during training. (default: ``True``)
+        ignore_metrics (List[str], optional): A list of regexes for metrics to ignore when logging. (default: ``None``)
```

Comment:
wdyt about just doing glob wildcards instead of regex. glob is a lot more user-friendly and probably sufficient?

Reply:
i don't have a strong opinion, ok with either.

Reply:
went ahead and switched to glob patterns